### PR TITLE
TAM CLI and YANG documentation

### PR DESCRIPTION
**Pull request by dutta-partha**
For :
   IFA
   Tail timestamping.

### CODE REVIEW

File: src/CLI/clitree/cli-xml/ifa.xml [hunk 1]
```diff
@@ -57,13 +95,46 @@ limitations under the License.
...
        <ACTION builtin="clish_pyobj">sonic_cli_ifa get_tam_ifa_flow_stats ${name}</ACTION>
+       <DOCGEN>
+            <DESCRIPTION>
+                Shows IFA flow statistics.
+            </DESCRIPTION>
+            <USAGE>
+                Use this command to display packet count and byte count for matcing IFA flow.
+            </USAGE>
+            <EXAMPLE summary="This">
+                sonic# show tam int-ifa statistics
```
Comment (asha-behera) [inline src/CLI/clitree/cli-xml/ifa.xml:106]: I think it is better to have examples with "show tam int-ifa statistics all" and "show tam-int-ifa statistics <flowname>

File: src/CLI/clitree/cli-xml/ifa.xml [hunk 2]
```diff
@@ -45,6 +60,29 @@ limitations under the License.
...
       </PARAM>
        <ACTION builtin="clish_pyobj">sonic_cli_ifa get_sonic_ifa_sonic_ifa_tam_int_ifa_flow_table show_tam_flow.j2 ${name}</ACTION>
+       <DOCGEN>
+            <DESCRIPTION>
+                Show IFA flow configuration.
+            </DESCRIPTION>
+            <USAGE>
+                Use this command to display IFA flow configuration.
+            </USAGE>
+            <EXAMPLE summary="This">
```
Comment (asha-behera) [inline src/CLI/clitree/cli-xml/ifa.xml:70]: Pls correct summary

File: src/CLI/clitree/cli-xml/ts.xml [hunk 3]
```diff
@@ -17,17 +17,86 @@
...
+                Shows IFA Tail Timestamping feature status.
+            </DESCRIPTION>
+            <USAGE>
+                After enabling IFA Timestamping feature, use this command to display feature status.		 
+            </USAGE>
+            <EXAMPLE summary="Shows IFA Tail Timestamping feature enabled status">
+                sonic(config)# tam
+                sonic(config-tam)# int-ifa-ts
+                sonic(config-int-ifa-ts)# feature enable
+
```
Comment (asha-behera) [inline src/CLI/clitree/cli-xml/ts.xml:31]: Delete unnecessary spaces

File: src/CLI/clitree/cli-xml/ts.xml [hunk 4]
```diff
@@ -17,17 +17,86 @@
...
       <ACTION builtin="clish_pyobj">sonic_cli_tam_int_ifa_ts get_sonic_tam_int_ifa_ts_sonic_tam_int_ifa_ts_tam_int_ifa_ts_flow_table show_tam_int_ifa_ts_flow.j2 ${name}</ACTION>
+       <DOCGEN>
+            <DESCRIPTION>
+                Shows IFA Tail Timestamping flow configutaion details.
+            </DESCRIPTION>
+            <USAGE>
+                Use this command to display flow configuration once flow configuration has been done.
+            </USAGE>
+            <EXAMPLE summary="Shows IFA Tail Timestamping flow details">
+                sonic# show tam int-ifa-ts flow
```
Comment (asha-behera) [inline src/CLI/clitree/cli-xml/ts.xml:49]: Same here, diff examples for all and individual flow is better

File: src/CLI/clitree/cli-xml/ts.xml [hunk 5]
```diff
@@ -17,17 +17,86 @@
...
+       <DOCGEN>
+            <DESCRIPTION>
+                Shows IFA Tail Timestamping flow statistics
+            </DESCRIPTION>
+            <USAGE>
+                Use this command once all required configurations are done and traffic is flowing.
+            </DESCRIPTION>
+            </USAGE>
+            <EXAMPLE summary="Shows IFA Tail Timestamping flow statistics">
+                sonic# show tam int-ifa-ts statistics
```
Comment (asha-behera) [inline src/CLI/clitree/cli-xml/ts.xml:92]: All possible examples, needs to be given here too

File: src/CLI/clitree/cli-xml/ts.xml [hunk 6]
```diff
@@ -125,7 +218,19 @@
...
               </PARAM>
 
-	   <ACTION builtin="clish_pyobj">sonic_cli_tam_int_ifa_ts patch_sonic_tam_int_ifa_ts_sonic_tam_int_ifa_ts_tam_int_ifa_ts_flow_table_tam_int_ifa_ts_flow_table_list ${flow-name} ${acl-table-name} ${acl-rule-name}</ACTION>
+           <ACTION builtin="clish_pyobj">sonic_cli_tam_int_ifa_ts patch_sonic_tam_int_ifa_ts_sonic_tam_int_ifa_ts_tam_int_ifa_ts_flow_table_tam_int_ifa_ts_flow_table_list ${flow-name} ${acl-table-name} ${acl-rule-name}</ACTION>
+           <DOCGEN>
+                <DESCRIPTION>
+                   Configure Tail Timestamping flow parameters.
+                </DESCRIPTION>
+                <USAGE>
+                   Use this command to configure Tail Timestamping flow table.
```
Comment (asha-behera) [inline src/CLI/clitree/cli-xml/ts.xml:227]: Should we give additional information regarding flow table ?

File: src/CLI/clitree/cli-xml/ts.xml [hunk 7]
```diff
@@ -125,7 +218,19 @@
...
+                </USAGE>
+                <EXAMPLE summary="Tail Timestamping flow configuration">
+                   dhcp-10-59-142-164(config-int-ifa-ts)# flow flow32 acl-table tacl1 acl-rule trule1
+                   dhcp-10-59-142-164(config-int-ifa-ts)#
```
Comment (asha-behera) [inline src/CLI/clitree/cli-xml/ts.xml:231]: We can do away with redundant empty prompt. to keep it clean

Comment (dutta-partha) [reply src/CLI/clitree/cli-xml/ts.xml:231]: Empty prompt kept to show that no message comes after running this command.

**Comment by dutta-partha** [comment]
This PR is no longer needed.

**Pull request closed without merging**